src/CompileErrorProcessor.ts

```diff
@@ -55,8 +52,8 @@ export class CompileErrorProcessor {
                     this.compilingLines = this.compilingLines.concat(newLines);
                     if (this.status === CompileStatus.compiling) {
                         //check to see if we've entered an error scenario
-                        let errors = this.getErrors();
-                        if (errors.length > 0) {
+                        let hasError = /\berror\b/gi.test(responseText);
```

Comment:
I don't have a full understanding of the scope of this code, but will this be incorrectly triggered if the response includes the term "error"? For instance, if there's a situation where `responseText` includes a line of source code and that code has a variabled named `error`?

Reply:
We're processing the compiling output only - this isn't processing runtime output.

That seemed wasteful to run the complete error detection but if you guys think it's unsafe I can revert.

Reply:
Also if it's a false positive no errors will be sent and it shouldn't stop the run process.

Reply:
Ok. Thanks for the explanation!